packages/blocks/src/api/factory.js

```diff
@@ -78,7 +78,7 @@ export function createBlock( name, attributes = {}, innerBlocks = [] ) {
 		{}
 	);
 
-	const clientId = uuid();
+	const clientId = id || uuid();
```

Comment:
Not sure I like this change, if  you're just editing an existing block, just edit it, don't use "createBlock"

Reply:
Yeah, I have the same feeling that's why I asked for input from your side :)